internal/gatewayapi/contexts.go

```diff
@@ -68,10 +68,11 @@ func (g *GatewayContext) attachEnvoyProxy(resources *Resources) {
 type ListenerContext struct {
 	*gwapiv1.Listener
 
-	gateway           *GatewayContext
-	listenerStatusIdx int
-	namespaceSelector labels.Selector
-	tlsSecrets        []*corev1.Secret
+	gateway                   *GatewayContext
+	listenerStatusIdx         int
+	namespaceSelector         labels.Selector
+	tlsSecrets                []*corev1.Secret
+	frontendValidationCACerts []byte
```

Comment:
do we need an extra field here ? it will cost more memory
can we directly assign `validateAndGetFrontendValidationCACerts` to the IR ?